src/main/scala/dsptools/numbers/chisel_base_type_extension/ChiselBaseNum.scala

```diff
@@ -9,7 +9,13 @@ object ChiselBaseNum {
 trait ChiselBaseNum[A <: Data] extends Any {
```

Comment:
Not sure about this name. This trait is really about power of two operations- maybe we should call it `BinaryRepresentation` or something that suggest that more.